gensrc/thrift/PlanNodes.thrift

```diff
@@ -315,6 +315,15 @@ struct TPaimonDeletionFile {
     3: optional i64 length
 }
 
+// refer to https://github.com/delta-io/delta/blob/master/PROTOCOL.md#deletion-vector-descriptor-schema
+struct TDeletionVectorDescriptor {
```

Comment:
Is iceberg dv going to reuse this field?

Reply:
Yes